keyboards/gmmk/pro/keymaps/default/keymap.c

```diff
@@ -16,6 +16,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #include QMK_KEYBOARD_H
 
+#define ______ KC_TRNS
+
```

Comment:
```suggestion
```
This alias already exists in core (albeit with seven underscores instead of 6, to maintain alignment with other keycodes: 2 char prefix, underscore, 4 char name)

Reply:
Thanks for the update there - I'm operating off some older approaches in the codebase, let me make that change and I'll update the PR. 😄